### PR TITLE
feat(contracts-ops): tokenRegistry fork tests

### PR DESCRIPTION
**Pull request by odyslam**
## PR Checklist

- [x] Added Tests
- [x] Updated Documentation
- [x] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/contracts-bridge/contracts/test/TokenRegistry.t.sol [hunk 1]
```diff
@@ -18,7 +18,7 @@ contract TokenRegistryTest is BridgeTestFixture {
     using TypedMemView for bytes29;
     using BridgeMessage for bytes29;
 
-    function setUp() public override {
+    function setUp() public virtual override {
```
Comment (prestwich) [inline packages/contracts-bridge/contracts/test/TokenRegistry.t.sol:21]: if it just invokes super, this should actually be deleted

File: packages/contracts-bridge/contracts/test/TokenRegistry.t.sol [hunk 2]
```diff
@@ -464,7 +471,10 @@ contract TokenRegistryTest is BridgeTestFixture {
         bytes32 id = "It's over 9000";
         address repr = tokenRegistry.getRepresentationAddress(domain, id);
         assertEq(repr, address(0));
-        address calculated = computeCreateAddress(address(tokenRegistry), 2);
+        address calculated = computeCreateAddress(
```
Comment (prestwich) [inline packages/contracts-bridge/contracts/test/TokenRegistry.t.sol:474]: should this be a named utility function? `expectedTokenAddress` or something like that?

File: packages/contracts-bridge/contracts/test/TokenRegistry.t.sol (hunk 1 above)
Comment (odyslam) [reply packages/contracts-bridge/contracts/test/TokenRegistry.t.sol:21]: Makes sense

File: packages/contracts-bridge/contracts/test/TokenRegistry.t.sol (hunk 2 above)
Comment (odyslam) [reply packages/contracts-bridge/contracts/test/TokenRegistry.t.sol:474]: TBH, I am not sold with adding all these utility functions in term of whether it makes the code more readable (albeit reducing code repeat).

Comment (prestwich) [reply packages/contracts-bridge/contracts/test/TokenRegistry.t.sol]: the reason I like utility functions is that it pushes the code towards even abstraction layers. When I'm reading a test, I don't want to break to remember how contract addresses are computed, or whether my registry uses `create` or `create2`. That is a different abstraction layer than the test functions on, and the details should be hidden when we're reading the test. Only things that work _on the same abstraction layer_ should be in the test

obv that's a bit of a muddy boundary, but clearly we're not interested in how addresses are calculated here

File: packages/contracts-bridge/contracts/test/TokenRegistry.t.sol [hunk 3]
```diff
@@ -490,10 +478,7 @@ contract TokenRegistryTest is BridgeTestFixture {
         vm.assume(domain != homeDomain && domain != 0);
         address repr = tokenRegistry.getRepresentationAddress(domain, id);
         assertEq(repr, address(0));
-        address calculated = computeCreateAddress(address(tokenRegistry), 2);
-        // test event emmission
-        vm.expectEmit(true, true, true, false);
-        emit TokenDeployed(domain, id, calculated);
+
```
Comment (prestwich) [inline packages/contracts-bridge/contracts/test/TokenRegistry.t.sol:481]: was this removed on purpose?

Comment (odyslam) [reply packages/contracts-bridge/contracts/test/TokenRegistry.t.sol:481]: The tests should not be green in this one. Checking.

**Review by prestwich — APPROVED** [review]

**Pull request merged**